src/components/themed/WalletListCurrencyRow.js

```diff
@@ -1,5 +1,6 @@
 // @flow
```

Comment:
Change commit message to say "Fix WalletListCurrencyRow.js functions to use bns operations"